index.html

```diff
@@ -57,7 +57,7 @@
 								<a target="_self" href="http://fossasia.org"><img alt="FOSSASIA Summit" class="logo logo-light" src="img/fossasia-long.png">
 								<img alt="FOSSASIA" class="logo logo-dark" src="img/fossasia-dark.png"></a>
 							</div>
-							
+
```

Comment:
Remove extra space from here.

Reply:
@harshit98 I had added that to visually segregate the two divs
Do you still want me to remove it?